docs/operator_installation_details.md

```diff
@@ -1,9 +1,28 @@
 # Install ClickHouse Operator
 
-Verify [clickhouse-operator-install-bundle.yaml][clickhouse-operator-install-bundle.yaml] file availability.
+# Prerequisites
+
+1. Kubernetes instance with the following version considerations:
+    1. `clickhouse-operator` versions **before** `0.16.0` requires [Kubernetes prior to `1.16`](https://kubernetes.io/releases/).
+    1. `clickhouse-operator` versions `0.16.0` **and after** require [Kubernetes version `1.23` and after](https://kubernetes.io/releases/).
```

Comment:
compatible with kubernetes version 1.22 and after

Reply:
Updated.

Reply:
oops, sorry my mislead

`v1beta1` for CRDs is deprecated on 1.16 and disabled on 1.22
so `clickhouse-operator 0.16+` should works with `kubernetes 1.16+` (not require)